checkpoint_ngfw/plugin.spec.yaml

```diff
@@ -473,6 +474,13 @@ connection:
     required: true
     example: true
     default: true
+  discard_other_sessions:
+    title: Discard Other Sessions
+    type: boolean
+    description: Discard all other user sessions. This can fix errors when objects are locked by other sessions
```

Comment:
I think this name could be more user friendly. 
This only happens on older versions of the firewall IIRC.
It's an API workaround, better to call it something like "Older than Rxx" but that name is still meh.

We should be more explicit here as well in the description and help.md on what this actually means.
discard other sessions translates to logging any existing users out of the Checkpoint UI, which means they could lose their sessions. We instituted this option as a workaround to an API bug in older versions of Checkpoint.

Reply:
I disagree...I want to be very clear that we are discarding sessions here. I don't want to rely on the user reading the description (they won't) and accidentally blowing away someone else working on the server.

Reply:
I think the `Discard Other Sessions` is too technical a language to get the point across. When I first looked at it,  I didn't understand what was happening.

@mrinehart-r7's  suggestion of "Force Changes" was better - and we could auto-detect and only apply it if the version is older. So if they're on a new version where it's not needed it won't be used.